Modix/ModixBot.cs

```diff
@@ -98,6 +115,8 @@ public void LoadConfig()
                 ReplToken = Environment.GetEnvironmentVariable("ReplToken"),
                 StackoverflowToken = Environment.GetEnvironmentVariable("StackoverflowToken"),
                 PostgreConnectionString = Environment.GetEnvironmentVariable("MODIX_DB_CONNECTION"),
+                DiscordClientId = Environment.GetEnvironmentVariable("DiscordClientId"),
+                DiscordClientSecret = Environment.GetEnvironmentVariable("DiscordClientSecret"),
             };
```

Comment:
Probably doesn't need to be done now, but i'd love to move the configuration over to Microsoft.Extensions.Configuration